packages/react-chat/src/components/Messages/MessageReactions.tsx

```diff
@@ -0,0 +1,69 @@
+import { BaseEmoji, Emoji } from "emoji-mart";
+import React from "react";
+import styled from "styled-components";
+
+import { ReactionButton } from "../Reactions/ReactionButton";
+
+interface MessageReactionsProps {
+  messageReactions: BaseEmoji[];
+  setMessageReactions: React.Dispatch<React.SetStateAction<BaseEmoji[]>>;
+}
+
+export function MessageReactions({
+  messageReactions,
+  setMessageReactions,
+}: MessageReactionsProps) {
+  return (
+    <Reactions>
+      {messageReactions.map((reaction) => (
+        <EmojiReaction
+          className={`${messageReactions.includes(reaction) && "chosen"}`}
```

Comment:
since you're iterating over messageReactions then messageReactions always includes reaction